src/Agent.Plugins/Agent.Plugins.csproj

```diff
@@ -19,7 +19,15 @@
     <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
     <PackageReference Include="System.Security.Cryptography.ProtectedData" Version="4.4.0" />
     <PackageReference Include="System.Text.Encoding.CodePages" Version="4.4.0" /> -->
-    <PackageReference Include="vss-api-netcore" Version="0.5.91-private" />
+    <PackageReference Include="ContentStoreInterfaces" Version="0.20181220.6-Dogfood" />
```

Comment:
why we need to do this? shouldn't the `vss-api-netcore` has dependency on `ContentStoreInterfaces`?, also why we are using `x-Dogfood` version? does it pass the OSS assets scan?

Reply:
I think we need an explicit PackageReference here for Agent.Plugins to be able to directly reference the types in ContentStoreInterfaces.  This was my intention.

ContentStoreInterfaces is built by the BuildXL team.  Previously we had forked their code as they were not yet build netstandard, but now they are so we deleted the fork to use their build directly.

I'll double check with them on the naming of the package and OSS asset scan. Good call!

Reply:
[chatted with Ting]
Does this work if we remove explicit references to RuntimeContracts and ContentStoreInterfaces? (and have them work implicitly via vss-api-netcore)

Reply:
@TingluoHuang  let me know what you think